test/ci/macos/run-ngraph-tf-mac-build.sh

```diff
@@ -223,7 +223,7 @@ if [ -z "${tf_dir}" ] ; then
     # If installing into the OS, use:
     # sudo --preserve-env --set-home pip install --ignore-installed ${PIP_INSTALL_EXTRA_ARGS:-} "${WHEEL_FILE}"
     # Here we are installing into a virtual environment, so DO NOT USE SUDO!!!
-    pip install -U tensorflow==1.12.0
+    pip install -U tensorflow
```

Comment:
Use the specific version of TensorFlow i.e., 1.13.1.

Reply:
OK, I locked the version used to 1.13.1 (i.e. "tensorflow==1.13.1")